src/display/views/help/components/__tests__/ContactFormPane.spec.js

```diff
@@ -10,28 +10,31 @@ const createTestProps = (propOverrides) => ({
 	...propOverrides
 });
 
+// The below test was for the old version when it was getting redirected as an email, now its getting redirected to github issues page as wanted.
```

Comment:
Since this doesn't need to be a form anymore, we can probably remove all the behavior tests in this file and just keep the snapshot test to make sure the component is rendering the way we want. 

To update the snapshot, uncomment the `describe('snapshots',` block and in the command line, run 

```
npm run test -- -u
```

This will pass the `-u` flag to the Jest test runner, which will instruct it to update the snapshots to reflect the new markup for the component.

Reply:
Ahhh, I see. I'll try to do that. Thank you for pointing this out!